test/examples_test.go

```diff
@@ -181,6 +181,18 @@ func getExamplePaths(t *testing.T, dir string) []string {
 			return filepath.SkipDir
 		}
 		if info.IsDir() == false && filepath.Ext(info.Name()) == ".yaml" {
+			// Ignore test matching the regexp in the TEST_EXAMPLES_IGNORES
+			// environement variable.
+			val, ok := os.LookupEnv("TEST_EXAMPLES_IGNORES")
+			if ok {
+				re := regexp.MustCompile(val)
```

Comment:
nit: it looks like this val check and re compilation could be moved up out of the filepath.Walk.

Reply:
I think if we move out of the `filepath.Walk` we are only getting the directories ? And we need to have that condition to filter only the yaml files : 

```go
if info.IsDir() == false && filepath.Ext(info.Name()) == ".yaml" {
```

Reply:
Ah right! My mistake. Sorry!